crowbar_framework/app/models/service_object.rb

```diff
@@ -1449,6 +1459,11 @@ def proposal_dependencies(role)
     []
   end
 
+  def skip_batch_for_node?(batch, node_name, old_role, role)
+    # By default dont skip anything
+    false
+  end
```

Comment:
I'd suggest adding some commit where we override this method for some of the core barclamps (ntp, dns & logging come to my mind since most of the time when reapplying, there's no change that impacts the client roles -- iirc, for dns there's only one attribute that has an impact)

Reply:
mmh, will that make this PR slower to review+merge? I rather do it on a separate one, or use the nova one as example instead than adding it to this and making the whole PR being blocked.

Reply:
Ok.